authui/src/authflowv2/base.css

```diff
@@ -0,0 +1,35 @@
+@layer base {
+  html,
+  body {
+    @apply m-0 p-0 h-full min-h-min flex flex-col;
```

Comment:
I suggest we move these to page_frame.html, on the element classname.

`m-0` and `p-0` are redundant because Tailwind's preflight.css has done this for us.

Reply:
I suggest we leave it in the css file so that it can be overridden if needed, let me remove the redundant classes